### PR TITLE
some style/grammar changes

### PR DESCRIPTION
**Pull request by tahabi**

### CODE REVIEW

File: ocfweb/about/templates/about/staff.html [hunk 1]
```diff
@@ -93,16 +93,16 @@
     <div class="ocf-logos">
         <div class="container">
             <div class="row">
-                <h2>Our lab and servers run free and open-source software.</h2>
+                <h2>Our lab and servers run free and open-source software, like:</h2>
```
Comment (tahabi) [inline ocfweb/about/templates/about/staff.html]: I think "such as:" would also be a good alternative to the period alone as well

File: ocfweb/about/templates/about/staff.html [hunk 2]
```diff
@@ -50,14 +50,14 @@
...
-                <h2>OCF staff help to provide cool services to the Berkeley community.</h2>
+                <h2>OCF staff help provide cool services to the Berkeley community.</h2>
             </div>
             <div class="row">
                 <div>
                     <div class="col-sm-4">
                         <p><img src="{% static 'img/about/web.png' %}" alt="web hosting" /></p>
                         <h3>Web &amp; Email Hosting</h3>
-                        <p>We provide free web and email hosting to our more than 40,000 members, both students and student groups.</p>
+                        <p>We provide free web and email hosting to more than 40,000 of our members, both students and student groups.</p>
```
Comment (matthew-mcallister) [inline ocfweb/about/templates/about/staff.html]: This changes the meaning. The fact is we _have_ more than 40 000 members, not that we provide web/email hosting to all of them.

Comment (tahabi) [inline ocfweb/about/templates/about/staff.html]: Makes sense, but the original phrasing still seems awkward. Is there a better way of expressing the original sentiment?

Comment (matthew-mcallister) [inline ocfweb/about/templates/about/staff.html]: Maybe "We provide free web and email hosting to more than 40,000 OCF members".

**Pull request merged**